Bottom: 6061949ea93ceec5ee787e93b486625c2fa6ce8c
Top:    89165446dc46374bd8200cc32b53d7ce32487ca4
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-21 02:01:27 +0000

Refresh of currently-pcb-user-can-draw-ar

---

diff --git a/src/crosshair.c b/src/crosshair.c
index aab9de1..c53216f 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -180,7 +180,6 @@ XORDrawAttachedArc (Coord thick)
 {
   ArcType arc;
   BoxType *bx;
-  Coord wx, wy;
   Angle sa, dir;
   Coord wid = thick / 2;
   Coord r;
@@ -205,7 +204,7 @@ XORDrawAttachedArc (Coord thick)
     {
       gui->graphics->draw_arc (Crosshair.GC, arc.X, arc.Y, r - wid, r - wid, sa, dir);
       gui->graphics->draw_arc (Crosshair.GC, bx->X1, bx->Y1, wid, wid, sa,      -180 * SGN (dir));
-      gui->graphics0>draw_arc (Crosshair.GC, bx->X2, bx->Y2, wid, wid, sa + dir, 180 * SGN (dir));
+      gui->graphics->draw_arc (Crosshair.GC, bx->X2, bx->Y2, wid, wid, sa + dir, 180 * SGN (dir));
     }
 }
